src/client/src/containers/Footer/index.tsx

```diff
@@ -105,15 +107,16 @@ class Footer extends React.Component<Props> {
       selectedFunctions,
       functions,
       vscode,
-      openPostGenModal
+      openPostGenModal,
+      intl
     } = this.props;
     e.preventDefault();
     // @ts-ignore
     vscode.postMessage({
       module: EXTENSION_MODULES.GENERATE,
       command: EXTENSION_COMMANDS.GENERATE,
       track: false,
-      text: "Sending generation info...",
+      text: intl.formatMessage(PAYLOAD_MESSAGES_TEXT.SENT_GENERATION_INFO_TEXT),
```

Comment:
I am not sure how would this affect but to be consistent with all the other strings, should we have the text as part of the `defineMessages`?

Reply:
Just wondering, should we be localizing strings in the payloads we send to the extension? Or is this a message that's displayed in the client?

Reply:
yeah I realized that too - I just changed it to not be localized